e2e/test/scenarios/sharing/downloads/downloads.cy.spec.js

```diff
@@ -160,10 +161,12 @@ describe("scenarios > dashboard > download pdf", () => {
   beforeEach(() => {
     restore();
     cy.signInAsAdmin();
+    cy.deleteDownloadsFolder();
```

Comment:
Can you please add it to other specs that use `verifyDownload`?

Reply:
I did a quick search for `"verifyDownload"` in the app and this test suite was the only place I found it